### DISCUSSION TITLE
How to use issue templates with "Reference in new issue"

### DISCUSSION THREAD

**Discussion by jonespm** [community/community#23034] [New to GitHub]
<p>I recently added issue templates to our repository and while they’re great, I’ve found that when I use the feature “Reference in new issue” of clicking on a line in the code to create an issue, it doesn’t seem to work with issue templates. Is there some special macro or something I need to add to my templates to get this to work? I looked through the docs but didn’t see anything. As a workaround, I’m using the “Copy permalink” and carrying that through to the issue. Thanks!</p>
<p><img src="https://user-images.githubusercontent.com/27447/181074281-d8d11de6-e93e-4af8-b34f-919c3f9e95d8.png" alt="image.png" data-base62-sha1="lqjIX4neydvhFhaAKMwRxC0N0Ja" width="620" height="478"></p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/jonespm">@jonespm</a>,</p>
<p>Thanks for taking the time to report this. I’ve been able to reproduce this and I’ve opened an issue for our team. I don’t have an ETA on a fix right now but this is in good hands.</p>

**Reply by tsuyoshinagaeda** [reply]
@AndreaGriffiths11 
「I use the feature “Reference in new issue” of clicking on a line in the code to create an issue, it doesn’t work with issue templates. 」

It seems that this issue has not yet been fixed.
We would like to use this feature to efficiently manage issues.
Could you please let us know when the fix is ​​scheduled?
